docs/keycodes.md

```diff
@@ -256,6 +256,7 @@ This is a reference only. Each group of keys links to the page documenting their
 |----------------------------------|---------|------------------------------------|
 |`MAGIC_SWAP_CONTROL_CAPSLOCK`     |         |Swap Caps Lock and Left Control     |
 |`MAGIC_CAPSLOCK_TO_CONTROL`       |         |Treat Caps Lock as Control          |
+|`MAGIC_SWAP_CONTROL_LGUI`         |         |Swap Left Control and GUI           |
```

Comment:
I think it would be better to use `LCTL` here (and in the other places), and add the corresponding stuff for the right side too.

Reply:
Will change, thanks